index.bs

```diff
@@ -885,16 +894,28 @@ To <dfn>serialize an [=attribution report=]</dfn> |report|, run the following st
 
 1. Let |destination| be |report|'s [=attribution report/attribution destination=].
 1. Assert: |destination| is not an [=opaque origin=].
+1. Let |falseTriggerProbability| be 0.
+1. If |report|'s [=attribution report/source type=] is "`event`", set |falseTriggerProbability|
+    to the user agent's [=event source falsely mode rate=].
+1. Let |triggerDataNoiseProbability| be the user agent's [=trigger data noise rate=].
+1. If |report|'s [=attribution report/source type=] is "`event`", set |triggerDataNoiseProbability|
+    to the user agent's [=event source trigger data noise rate=].
 1. Let |data| be a [=map=] of the following key/value pairs:
 
     : "`attribution_destination`"
     :: |destination|, <a href="https://html.spec.whatwg.org/multipage/origin.html#serialization-of-a-site">serialized</a>
+    : "`false_trigger_probability`"
+    :: |falseTriggerProbability|
     : "`source_type`"
     :: |report|'s [=attribution report/source type=]
     : "`source_event_id`"
     :: |report|'s [=attribution report/event id=], [=serialize an integer|serialized=]
+    : "`source_event_id_noise_probability`"
+    :: 0
     : "`trigger_data`"
     :: |report|'s [=attribution report/trigger data=], [=serialize an integer|serialized=]
+    : "`trigger_data_noise_probability`"
+    :: |triggerDataNoiseProbability|
```

Comment:
One caveat with this approach is that the probabilities may be wrong for reports in storage that haven't been sent yet, e.g. if the browser is updated between the time a source is stored and the time the report is sent.

We could add fields representing the probabilities to the `attribution source` and `attribution trigger` structures, but it would incur storage overhead. Thoughts?